package.json

```diff
@@ -14,9 +14,9 @@
   ],
   "main": "dist/klaro.js",
   "scripts": {
-    "make-dev": "APP_ENV=dev APP_DEV_MODE=server webpack-dev-server --config webpack.config.js --history-api-fallback",
-    "make-watch": "APP_ENV=dev APP_DEV_MODE=watch webpack --watch --config webpack.config.js",
-    "make": "APP_ENV=production APP_VERSION=`git tag --points-at HEAD` APP_COMMIT=`git rev-parse HEAD` webpack --config webpack.config.js"
+    "make-dev": "cross-env-shell \"APP_ENV=dev APP_DEV_MODE=server webpack-dev-server --config webpack.config.js --history-api-fallback\"",
```

Comment:
Can you remove '/"' and  '"/' because of this its not working in my windows for all three statement